addons/advanced_fatigue/initSettings.sqf

```diff
@@ -4,7 +4,9 @@
     [LSTRING(Enabled), LSTRING(Enabled_Description)],
     LSTRING(DisplayName),
     true,
-    true
+    true,
+    {[QGVAR(enabled), _this] call EFUNC(common,cbaSettings_settingChanged)},
+    true // Needs mission restart
```

Comment:
this comment looks quite weird in each file because other params are not commented

Reply:
TBH I would prefer the other `true`'s to be commented too. But on the other end we cannot comment the same things over and over again everywhere.
Or better we probably shouldn't.